browses.md

```diff
@@ -54,6 +54,27 @@ Error codes:
 
 **404** if the browse index doesn't exist
 
+## Browse indexes configured for metadata linking
+### Get a single index configured to render given field as a link
+**/api/discover/browses/search/byField?field=<:field>**
+
+Example: <http://dspace7.4science.it/dspace-spring-rest/#http://dspace7.4science.it/dspace-spring-rest/api/discover/browses/search/byField/dc.contributor.author>
+
+Provide detailed information about a specific browse index and access to the list of items and entries in the index. The JSON response document is as the previous section.
+
+The supported parameters are:
+* field: The metadata field name, eg. dc.contributor.author
+
+Error codes:
+**404** if the browse index is not a metadata browse or the browse index doesn't exist at all
```

Comment:
a search method should never return 404, it could eventually return a 204 No content.
See for instance https://github.com/DSpace/RestContract/blob/main/workspaceitems.md#get-single-workspace-item-from-item-uuid for a similar scenario

Reply:
Done! And included in ITs as well as a 400 Bad Request test